db/schema.rb

```diff
@@ -10,15 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_07_095906) do
+ActiveRecord::Schema.define(version: 2020_09_08_110952) do
 
   create_table "bookings", force: :cascade do |t|
     t.integer "user_id"
     t.integer "host_id"
     t.datetime "start_time"
     t.datetime "end_time"
     t.datetime "confirmed_at"
-    t.integer "confirmed_by"
+    t.integer "confirmed_by_id"
```

Comment:
I've migrated this column to allow us to pass users directly to the confirmed_by field without the need to call the users id.

```
booking.update(confirmed_by: user)
```
rather than

```
booking.update(confirmed_by: user.id)
```